front/app/components/ProjectCard/LargeProjectCard.tsx

```diff
@@ -0,0 +1,333 @@
+import React, { memo } from 'react';
+
+import {
+  Icon,
+  Box,
+  media,
+  colors,
+  fontSizes,
+  defaultCardStyle,
+  defaultCardHoverStyle,
+  isRtl,
+} from '@citizenlab/cl2-component-library';
+import { isEmpty } from 'lodash-es';
+import { RouteType } from 'routes';
+import styled, { useTheme } from 'styled-components';
+
+import usePhase from 'api/phases/usePhase';
+import usePhases from 'api/phases/usePhases';
+import { getInputTerm } from 'api/phases/utils';
+import useProjectImage from 'api/project_images/useProjectImage';
+import { CARD_IMAGE_ASPECT_RATIO } from 'api/project_images/useProjectImages';
+import useProjectById from 'api/projects/useProjectById';
+import { getProjectUrl } from 'api/projects/utils';
+
+import AvatarBubbles from 'components/AvatarBubbles';
+import FollowUnfollow from 'components/FollowUnfollow';
+import T from 'components/T';
+
+import Link from 'utils/cl-router/Link';
+
+import ProjectHeader, {
+  ContentHeaderBottomMargin,
+  ContentHeaderHeight,
+} from './ProjectHeader';
+import ScreenReaderContent from './ScreenReaderContent';
+import {
+  handleProjectCardOnClick,
+  handleProjectTitleOnClick,
+  ProjectImage,
+  ProjectTitle,
+} from './utils';
+
+const Container = styled(Link)`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 25px;
+  position: relative;
+  cursor: pointer;
+  align-items: stretch;
+  justify-content: space-between;
+
+  ${defaultCardStyle};
+
+  ${isRtl`
+    flex-direction: row-reverse;
+  `}
+
+  ${media.desktop`
+    ${defaultCardHoverStyle};
+  `}
+`;
+
+const ProjectImageContainer = styled.div`
+  width: 50%;
+  display: flex;
+  aspect-ratio: ${CARD_IMAGE_ASPECT_RATIO} / 1;
+  margin-right: 10px;
+  overflow: hidden;
+  position: relative;
+  border-top-left-radius: 4px;
+  border-bottom-left-radius: 4px;
+
+  ${media.phone`
```

Comment:
It seems like it's impossible to encounter this large project card on a phone, so this can probably be deleted